utils/_context/_scenarios/__init__.py

```diff
@@ -469,6 +469,13 @@ def all_endtoend_scenarios(test_object):
         doc="Scenario with custom headers for DD_TRACE_HEADER_TAGS that libraries should reject",
     )
 
+    tracing_config_default = EndToEndScenario("TRACING_CONFIG_DEFAULT", doc="")
+    tracing_config_custom_server_error_statuses = EndToEndScenario(
+        "TRACING_CONFIG_CUSTOM_SERVER_ERROR_STATUSES",
+        weblog_env={"DD_HTTP_SERVER_ERROR_STATUSES": "200-201,202"},
```

Comment:
To reduce the number of scenario, and also to have this test activated by default on all dd-trace repo, you can use this on the default scenario: `"DD_HTTP_SERVER_ERROR_STATUSES": "250-251,252"`, and run your test on those status that are not used elsewhere.

Reply:
Unfortunately we cannot test this configuration in the default case because then the feature's default behavior is completely overriden. I did move the default behavior into the DEFAULT scenario so now we only have one additional scenario.

Also, for the custom DD_HTTP_SERVER_ERROR_STATUSES config, I renamed the scenario name to TRACING_CONFIG_NONDEFAULT so that when we add more tests to validate tracing configuration feature-flags, we can include them into this one scenario. This was done in 6476955acf0a9790c7b1095fc6f567699c6683b4